TestCases/vandv/rans/30p30n/config.cfg

```diff
@@ -0,0 +1,94 @@
+% ------------------------------- SOLVER -------------------------------- %
```

Comment:
Why no header? I personally dont mind having a infos about the origin of a case

Reply:
https://su2code.github.io/vandv/30p30n/